src/plone/restapi/tests/test_dxcontent_serializer.py

```diff
@@ -88,8 +89,10 @@ def setUp(self):
         self.portal.doc1.modification_date = DateTime("2015-04-27T10:24:11+00:00")
         IMutableUUID(self.portal.doc1).set("30314724b77a4ec0abbad03d262837aa")
 
-    def serialize(self):
-        serializer = getMultiAdapter((self.portal.doc1, self.request), ISerializeToJson)
+    def serialize(self, obj=None):
+        if obj is None:
```

Comment:
Can you elaborate why this is required? Are you aware that by doing this you are conditioning the rest of the tests in this TestCase?

Reply:
This method is hardcoded to `self.portal.doc1` and I need to serialize other objects.

Sure, it is intended to have this condition because all other tests expect to be this object.